packages/flutter/test/material/popup_menu_theme_test.dart

```diff
@@ -225,8 +225,8 @@ void main() {
         child: Column(
           children: <Widget>[
             Padding(
-              // The padding makes sure the menu as enough space to around to
-              // get properly aligned when displayed (`_kMenuScreenPadding`).
+              // The padding makes sure the menu has enough space to around it
```

Comment:
same

Reply:
Same, I removed it in https://github.com/flutter/flutter/pull/111060/commits/4d310b3c6f6d18f1f58b211ea0940fd907a281b0